src/hato/multipart.clj

```diff
@@ -55,12 +84,94 @@
        (take 30)
        (apply str "hatoBoundary")))
 
-(defn body
-  "Returns an InputStream from the multipart inputs.
+(defprotocol MultipartParam
+  (input-stream [content opts])
+  (length [content opts]))
+
+(extend-protocol MultipartParam
+  String
+  (input-stream [^String content opts]
+    (io/input-stream (.getBytes content (charset-encoding opts))))
```

Comment:
We need to make a best guess to determine the charset for the string so we look at either the :content-type string/map and parse it.